third_party/libwebp/BUILD.gn

```diff
@@ -110,7 +113,8 @@ config("libwebp_direct_config") {
       ]
       defines += [ "WEBP_HAVE_AVX2" ]
     }
-    if (current_cpu == "arm" || current_cpu == "arm64") {
+    if ((current_cpu == "arm" || current_cpu == "arm64") && arm_use_neon &&
+        (target_os == "android" || target_os == "linux")) {
```

Comment:
Use `is_android` and `is_linux` instead of explicitly using `target_os`. Also isn't checking if the `current_cpu` is arm/arm64 redundant if we're doing the same when setting `arm_use_neon`? It seems to me we should be doing one or the other

Reply:
You are probably right here. I looked at bad examples and added a worse one - that fixed the builds for now. I'll try take another pass of cleaning it up in more places than this.